### DISCUSSION TITLE
Using Tailwind CSS

### DISCUSSION THREAD

**Discussion by jasonleex** [xshapira/django-simple-svg#5] [Q&A] [answered]
As a beginner in Django and web development, can I use Tailwind CSS with `django-simple-svg`?

**Comment by xshapira** [comment, marked as answer]
Absolutely! You can use Tailwindcss in the same way you would use any other CSS classes. Here's a simple example from one of my projects:
```html
<li class="flex items-center mb-2">
   {% svg "icon-check" class="w-3 h-3 fill-current text-green-500 mr-3 flex-shrink-0" %}
</li>
```

Note that the `icon-check` should be stored in your SVGs folder named `svg` at the root of your static file directory.
